src/types/components/enhanced-table/enhancedTable.interface.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeEvent, MouseEventHandler } from 'react'
+import { Sort } from '~/types/common/common.index'
+
+export interface TableColumn {
+  field: string
+  calculatedCellValue?: () => string
+}
+
+export interface TableRowAction {
+  label: string
+  func: (id: string) => Promise<void>
+}
+
+export interface TableItem {

```

Comment:
remove it item should be generic

Reply:
i have added generic that extends TableItem that has "_id" property cause lint says that there is not "_id" in generic cause it is any type

Reply:
it`s bad choice because not all items in the table could have _id field, you should investigate and fix that error in other way

Reply:
yeap, i think so too
but as i see in the related components to this table, are using _id property for keys and some functions so it can't be omitted